packages/client/hmi-client/src/page/data-explorer/components/tera-search-results-list.vue

```diff
@@ -63,15 +63,17 @@
 
 <script setup lang="ts">
 import { isEmpty } from 'lodash';
-import { computed, PropType, ref } from 'vue';
-import type {
+import { computed, PropType, Ref, ref } from 'vue';
+import {
 	AddDocumentAssetFromXDDResponse,
+	AssetType,
+	type Dataset,
 	Document,
 	DocumentAsset,
+	Model,
 	ProjectAsset,
 	XDDFacetsItemResponse
 } from '@/types/Types';
```

Comment:
```suggestion
import type {
	AddDocumentAssetFromXDDResponse,
	AssetType,
	Dataset,
	Document,
	DocumentAsset,
	Model,
	ProjectAsset,
	XDDFacetsItemResponse
} from '@/types/Types';
```

Reply:
putting type here causes other issues, but, removing it seems to fix it.